packages/paste-website/src/pages/components/index.mdx

```diff
@@ -21,8 +24,28 @@ import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
 
 The components listed here are on our roadmap. Only a few of them are available today.
 
-If you’re planning on using an alpha or beta component, please check in with the Paste team through the <strong>#help-design-system</strong> channel on Slack.
-
+<Grid>
+  <Column span={8}>
+    <Card>
+      <Heading as="h4" variant="heading40">
+        Component Status Details
+      </Heading>
+      <Text as="p" marginBottom="space20">
+        <strong>Alpha</strong>: Component is a candidate for testing in products with guidance from the design systems
+        team. If you're planning on using one of these components, check in with the Paste team first through{' '}
+        <strong>#help-design-system</strong> in Slack.
+      </Text>
+      <Text as="p" marginBottom="space20">
+        <strong>Beta</strong>: Component is a candidate for prototypes and beta products. If you're planning on using
+        one of these components, check in with the Paste team first through <strong>#help-design-system</strong> in
+        Slack.
+      </Text>
+      <Text as="p" marginBottom="space20">
+        <strong>Production</strong>: We shipped a component! Everyone should use it.
```

Comment:
I'm still concerned with the positioning of this text.

The point of the lifecycle was to enable us to be a little more nimble and not have to jump through all the hoops to release a component to production (actual product, end user facing). This was because we're a small team, and having to design, spec, build, internal usability test, external usability test and go through UX review before  anyone would place a component in production is an unrealistic expectation. 

Given that we version our components and work iteratively the idea of the lifecycle was to allow us to punt customer usability and ux review steps to a later date, but people feel comfortable still using the component in products before hand. Stating that we'll get to it, and it's perfectly safe to use, embracing the idea of a design system evolving component designs over time as we learn.

But the way this reads still really, strongly indicates that components aren't "ready" until they are "production". Given the amount of work that entails based on the life cycle that's going to slow us down and adoption down.